tracer/test/test-applications/security/Samples.Security.AspNetCore5/Controllers/CreateExtraServiceController.cs

```diff
@@ -0,0 +1,25 @@
+﻿#nullable enable
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Samples.Security.AspNetCore5.Models;
+using Samples.Security.AspNetCore5.IdentityStores;
+using Datadog.Trace;
+
+namespace Samples.Security.AspNetCore5.Controllers;
+
+public class CreateExtraServiceController: Controller
+{
+    [HttpGet]
+    public IActionResult Index(string? serviceName)
+    {
+        using var scope = Tracer.Instance.StartActive("create-extra-service");
+        scope.Span.ServiceName = serviceName;
+
+        // fake a http.url tag so it'll appear in the snapshot
+        scope.Span.SetTag("http.url", "http://localhost:00000/createextraservice/?serviceName=extraVegetables");
```

Comment:
Just curious, why the word extraVegetables is hardcoded in this line instead of using the one coming from serviceName?

Reply:
just an oversite, does affect current tests, but guess it could cause problems if someone uses the end point in other tests, so fixed in 4b821b36d